assets/scripts/libraries/cookies-opt-in.js

```diff
@@ -374,16 +374,17 @@ var docCookies = {
         var cookieMessageInner = document.createElement('div');
         cookieMessageInner.classList.add('cookie-message__inner', 'govuk-width-container');
         cookieMessageInner.classList.add('site-container');
-        cookieMessageInner.innerHTML = '<div class="cookie-message__intro"><p>We use cookies to collect information about how you use crowncommercial.gov.uk. We use this information to make the website work as well as possible and improve government services.</p></div>';
+        cookieMessageInner.innerHTML = '<div class="cookie-message__intro"><p>We use non-essential cookies to help us improve this website and our services and any data collected is anonymised. Are you happy to help us improve our website and services by allowing us to use cookies on this website?</p></div>';
```

Comment:
Sorry, should have clarified this. Following what GDS does, I don't think we are looking to change the wording, let's confirm with Nicola

Reply:
ah ok no problem. I was just going by this document [https://docs.google.com/drawings/d/11SrPNqxxoGIHNJzs8erQRvj4SVLRIu10il7QyGmNw48/edit](url)